ml-agents-envs/mlagents/envs/brain.py

```diff
@@ -84,28 +84,28 @@ def merge_memories(m1, m2, agents1, agents2):
         return np.append(m1, m2, axis=0)
 
     @staticmethod
-    def process_pixels(image_bytes, gray_scale):
+    def process_pixels(image_bytes, gray_scale) -> np.ndarray:
```

Comment:
Can you add type annotations for the input args too?

Reply:
👍